components/automate-gateway/handler/ssoconfig.go

```diff
@@ -72,6 +69,54 @@ func (a *SsoConfig) GetSsoConfig(ctx context.Context, in *empty.Empty) (*sso.Get
 	}, nil
 }
 
+func (a *SsoConfig) DeleteSsoConfig(ctx context.Context, in *empty.Empty) (*sso.DeleteSsoConfigResponse, error) {
+	err := a.validateDeploymentType(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Printf("Received request to delete sso config")
+	res, err := a.getConfigData(ctx)
+
+	if err !=nil {
+		return nil, err
+	}
+	
+	if res.Config.Dex != nil {
+		url, err := getBastionUrl()
+		if err != nil {
+			return nil, err
+		}
+		fileName := "revert-status.txt"
+		err = ioutil.WriteFile(ssoFilesPath+fileName, []byte("Pending"), 0777)
+		if err != nil {
+			log.Printf("Unable to write file:", err)
+			return nil, err
+		}
+		go makeRequest("DELETE", *url, nil, fileName)
+		return &sso.DeleteSsoConfigResponse{
+            Message: "Started Disabling SSO Configuration",
+        }, nil
+    }
+
+	return &sso.DeleteSsoConfigResponse{
+		Message: "SSO Configuration not disabled successfully",
```

Comment:
Please change the message to Dex config not present or SSO config not present

Reply:
Fixed